### PR TITLE
github-runner: 2.279.0 -> 2.280.1

### PR DESCRIPTION
**Pull request by Fresheyeball**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by emmanuelrosa** [comment]
A more precise title would be helpful.

File: nixos/modules/services/continuous-integration/github-runner.nix [hunk 1]
```diff
@@ -120,13 +120,7 @@ in
...
       };
 
-      path = (with pkgs; [
-        bash
-        coreutils
-        git
-        gnutar
-        gzip
-      ]) ++ [
+      path = [
```
Comment (Mic92) [inline nixos/modules/services/continuous-integration/github-runner.nix:123]: It's not clear to me why those have been removed.

**Comment by veehaitch** [comment]
Result of `nixpkgs-review pr 133698` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>github-runner</li>
  </ul>
</details>

File: nixos/modules/services/continuous-integration/github-runner.nix (hunk 1 above)
Comment (veehaitch) [reply nixos/modules/services/continuous-integration/github-runner.nix:123]: While those packages are not strictly necessary to run the GitHub Actions Runner, they are the bare minimum for the [checkout action](https://github.com/actions/checkout).

**Comment by ajs124** [comment]
These commit message most certainly don't fit our CONTRIBUTING.md

Plus they're just useless.

**Comment by antifuchs** [comment]
Just a heads-up that [2.280.3](https://github.com/actions/runner/releases/tag/v2.280.3) is already out and due to the self-update/refusal to run older versions, 2.280.1 will arrive pre-broken (see #120263) /:

File: nixos/modules/services/continuous-integration/github-runner.nix (hunk 1 above)
Comment (SuperSandro2000) [reply nixos/modules/services/continuous-integration/github-runner.nix:123]: This would mean that basically everything is broken, right?

Comment (veehaitch) [reply nixos/modules/services/continuous-integration/github-runner.nix:123]: Not necessarily. You might be using the runner without the checkout action—I just don't think this is a common use case. If we leave those packages out, everybody else has to figure out the packages required to use the checkout action themselves. I'd definitively keep them.

**Pull request closed without merging**